docker/package/tezos_setup_wizard.py

```diff
@@ -815,8 +824,7 @@ def import_baker_key(self):
                         ):
                             self.query_step(
                                 get_ledger_derivation(
-                                    ledger_derivations,
-                                    base_ledger_url,
+                                    ledgers_derivations,
```

Comment:
I'm not sure how this works. IIUC, it will still only show all the derivation paths together like right now, except without specifying the accompanying base ledger url. So there is a bit less confusion, but it's still difficult to gather which ledger we are chooding. Can we not ask to choose the ledger url (if there is more than one ledger) in any case before this step and show only the relevant ones?
Maybe I missed something in the workflow, in that case I'd like to see some screenshots as I can't test this myself 😞

Reply:
So, currently, it shows 4 derivations for each ledger:
![image](https://user-images.githubusercontent.com/14182218/151587802-2a892608-18e9-460b-9d64-7de83b8d568d.png)
When you choose `Specify derivation path`, it asks for a specific ledger URL before querying the derivation path on it:
![image](https://user-images.githubusercontent.com/14182218/151587941-b50ca926-bff1-4ebe-ad8a-9f0a8b162f76.png)

Reply:
Ah, OK, I see my mistake :facepalm: Not sure how readable this is though, especially with the lack of column alignment :thinking: I still think it might be better to choose the ledger beforehand, but idk

Reply:
I agree, that it isn't very readable :unamused:

Perhaps, we can store a `dict` instead of a list and display derivations for each ledger separately :thinking:

Reply:
Unfortunately, I don't have access to the second ledger at the moment, but this is how it looks when we show derivations without showing the full ledger path:
![image](https://user-images.githubusercontent.com/14182218/151768033-e80f172f-7f64-41b6-9951-4f39b31dd786.png)

Reply:
Dividing the options per-ledger doesn't look bad to me.

However, I see that `Specify derivation path` is given separately, which makes me think there will be only one regardless of the number of ledgers (as in the screenshot above as well), but shouldn't it also be given per-ledger instead?

Reply:
`Specify derivation path` lets you specify the ledger. See the screenshot in my previous comment

Reply:
oh sorry, I completely missed that, never mind my previous comment :ok_hand:

Reply:
I've added proper alignment for cases when there are more than 10 options. Now it looks like this:
![image](https://user-images.githubusercontent.com/14182218/154258609-17220cf5-5cb1-45e4-b8d5-c80285369755.png)

Reply:
@ixahedron, @pasqu4le, may I ask you to re-review now?